guest/api/types.go

```diff
@@ -97,3 +113,8 @@ type NodeInfo interface {
 
 	Node() proto.Node
 }
+
+// PostFilterPlugin use nodeToStatusMap
+type NodeToStatusMap interface {
+	Map() map[string]StatusCode
```

Comment:
It's better to be:

```suggestion
// PostFilterPlugin use nodeToStatusMap
type NodeToStatus interface {
	Map() map[string]StatusCode
```

---

Also, you missed the comment change suggested on https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/pull/75#discussion_r1403489824

Reply:
Thanks, fixed this comment and your previous comment.